assets/javascripts/modules/analytics/ga.js

```diff
@@ -6,6 +6,8 @@ define(['modules/analytics/analyticsEnabled',
     'use strict';
 
     var _EVENT_QUEUE = [];
+    var experienceIsSet = 'stripeCheckout' in guardian;
```

Comment:
I think these should go into init() as that's guaranteed to be called after domcontentloaded, not executed at parse time like this function might be. There might be a race condition if other javascript set 'stripeCheckout' (even though Scala sets it in the template)